mmv1/third_party/terraform/resources/resource_bigtable_instance.go

```diff
@@ -41,7 +41,7 @@ func resourceBigtableInstance() *schema.Resource {
 				Type:        schema.TypeString,
 				Required:    true,
 				ForceNew:    true,
-				Description: `The name (also called Instance Id in the Cloud Console) of the Cloud Bigtable instance.`,
+				Description: `The name (also called Instance Id in the Cloud Console) of the Cloud Bigtable instance. It has to be 6-33 characters and must only contain hyphens, lowercase letters and numbers.`,
```

Comment:
Can we change "It has to be" to "Must be" here and elsewhere?

Reply:
Done.